website/docs/reference/source-configs.md

```diff
@@ -8,7 +8,17 @@ import ConfigGeneral from '/snippets/_config-description-general.md';
 
 ## Available configurations
 
-Sources only support one configuration, [`enabled`](/reference/resource-configs/enabled).
+<VersionBlock lastVersion="1.8">
+
+Sources supports [`enabled`](/reference/resource-configs/enabled) and [`meta`](/reference/resource-configs/meta).
```

Comment:
currently the doc mentions source only supports enabled. but other pages show that it supports meta, and not event_time.